packages/pretty-format/src/index.ts

```diff
@@ -27,7 +27,12 @@ const toString = Object.prototype.toString;
 const toISOString = Date.prototype.toISOString;
 const errorToString = Error.prototype.toString;
 const regExpToString = RegExp.prototype.toString;
-const symbolToString = Symbol.prototype.toString;
+const symbolToString =
+  typeof Symbol === 'function'
+    ? Symbol.prototype.toString
+    : function(this: symbol) {
```

Comment:
Is it possible for the alternative function to be called in an environment in which `Symbol` is not defined?

If not, what do you think about moving all the logic into `printSymbol` and follow example of `printBigInt`

Off the top of my head, I am not sure what is the purpose of `SYMBOL_REGEXP`

Reply:
> Is it possible for the alternative function to be called in an environment in which Symbol is not defined?

I don't think so. The alternate is constructed just so it matches the signature of the actual function and typescript won't complain.

I'll check out how you handle BigInt.

Reply:
After reading [Symbol Coercion](https://leanpub.com/understandinges6/read#leanpub-auto-symbol-coercion) in Understanding ES6 by Nicholas C. Zakas and experimenting to make sure that you cannot override the `toString` method, here is my suggestion:

```js
function printSymbol(val: symbol): string {
  return String(val).replace(SYMBOL_REGEXP, 'Symbol($1)');
}
```